be/src/exprs/runtime_filter.cpp

```diff
@@ -215,4 +231,15 @@ bool JoinRuntimeFilter::check_equal(const JoinRuntimeFilter& rf) const {
     return true;
 }
 
+void JoinRuntimeFilter::clear_bf() {
+    if (_num_hash_partitions == 0) {
+        _bf.clear();
+    } else {
+        for (size_t i = 0; i < _num_hash_partitions; i++) {
+            _hash_partition_bf[i].clear();
```

Comment:
need clear `_hash_partition_bf`?

Reply:
> need clear `_hash_partition_bf`?

we can't do this because serialize may rely on _hash_partition_bf's  size.

```cpp
size_t JoinRuntimeFilter::serialize(int serialize_version, uint8_t* data) const {
    size_t offset = 0;
#define JRF_COPY_FIELD(field)                     \
    memcpy(data + offset, &field, sizeof(field)); \
    offset += sizeof(field);
    JRF_COPY_FIELD(_has_null);
    JRF_COPY_FIELD(_size);
    JRF_COPY_FIELD(_num_hash_partitions);
    JRF_COPY_FIELD(_join_mode);
#undef JRF_COPY_FIELD

    if (_num_hash_partitions == 0) {
        offset += _bf.serialize(data + offset);

    } else {
        for (const auto& bf : _hash_partition_bf) {
            offset += bf.serialize(data + offset);
        }
    }
    return offset;
}```